src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/chapterPage/viewModel/ChapterPageViewModel.kt

```diff
@@ -0,0 +1,53 @@
+package org.wycliffeassociates.otter.jvm.app.ui.chapterPage.viewModel
+
+import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.subjects.PublishSubject
+import javafx.collections.FXCollections
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.ChapterPageModel
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.Verse
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import tornadofx.*
+
+class ChapterPageViewModel : ViewModel() {
+    val model = ChapterPageModel()
+    var activeChapter = PublishSubject.create<Int>()
+    var selectedTab = PublishSubject.create<String>()
+    val chapters = FXCollections.observableArrayList<String>()!!
+    val verses = FXCollections.observableArrayList<Verse>()!!
+    val bookTitle = model.bookTitle
+
+    fun changeContext(context: String) {
+        selectedTab.onNext(context)
+    }
```

Comment:
With the new enum, this can become `fun changeContext(context: Contexts)` and `var selectedTab = PublishSubject.create<Contexts>()`. That would keep you from needing to do `.label` in the `when` statement in the view